stock_picking_mixin/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# © 2021 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+{
+    "name": "Stock Picking Mixin",
```

Comment:
Preventing usage for other purposes, I think is better renaming it
```suggestion
    "name": "Stock Picking Mixins",
```

and, then rename the whole addon as `stock_picking_mixins`.